TSC-Charter.md

```diff
@@ -146,7 +145,7 @@ basis as defined in the development process.
 Collaborators may opt to elevate significant or controversial
 modifications, or modifications that have not found consensus to the TSC
 for discussion by assigning the `tsc-agenda` tag to a pull request or
-issue. The TSC voting members should serve as the final arbiter where required.
+issue. TSC votes should serve as the final arbiter where required.
```

Comment:
I think this paragraph should be left as-is.

Reply:
+1

Reply:
Would the following work better for everyone?
```suggestion
issue. The TSC should serve as the final arbiter where required.
```

Reply:
I don't think we are in agreement with the change. The TSC voting members _are_ the final arbiter.

Reply:
I think leaving it as "TSC voting members" removes ambiguity. I think characterizing it as the TSC as a whole is friendly, but not precise. I'd favor precision in this case.

Reply:
Maybe it's because I don't interpret English as you'all, but to me it adds some ambiguity: it could mean that I, as a TSC voting member, can serve as the final arbiter. But if we use "the TSC", it's very clear that we talk about the collective, not the individuals.
Would Gireesh suggestion (https://github.com/nodejs/TSC/pull/1360#discussion_r1145729795) work for everyone? Or does someone else has a suggestion? Or should I just shut up about it already 😅?

Reply:
Ah, I understand where you are coming from @aduh95. While English is not my native language either, I think `the TSC` is not precise.

I think something like the following could work:

> When consensus cannot be reached, the TSC voting members should serve as the final arbiter by casting a vote.

Wdyt?

Reply:
I love it, and I agree your suggestion in indeed less ambiguous.
```suggestion
issue. When consensus cannot be reached, the TSC voting members should serve as the
final arbiter by casting a vote.
```